conftest.py

```diff
@@ -0,0 +1,30 @@
+"""Place Pytest fixtures to be used across all tests here."""
+import pytest
+import ray
+
+
+@pytest.fixture(scope='session', autouse=True)
+def setup_ray_local():
+    """Initializes Ray in localmode before session starts."""
+    if not ray.is_initialized():
+        ray.init(memory=52428800,
+                 object_store_memory=78643200,
+                 ignore_reinit_error=True,
+                 log_to_driver=False)
+
+
+@pytest.fixture(scope='session', autouse=True)
+def cleanup_ray(request):
```

Comment:
Is there a reason you didn't use the clean-up pattern recommended by the pytest developers?

https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code